src/app/authentication/sign-in/sign-in.component.ts

```diff
@@ -6,7 +6,7 @@ import {NgForm} from '@angular/forms';
 @Component({
   selector: 'app-sign-in',
   templateUrl: './sign-in.component.html',
-  styleUrls: ['./sign-in.component.scss']
+  styleUrls: ['./sign-in.component.scss', '../authentication.component.scss']
```

Comment:
Why are there styles from another component? That should be encapsulated for one scss file per component.